R/zzx-deprecated.R

```diff
@@ -590,19 +590,45 @@ cdm_insert_zoomed_tbl <- function(dm, new_tbl_name = NULL, repair = "unique", qu
   dm_wo_outgoing_fks <-
     dm %>%
     update_filter(old_tbl_name, list_of(old_filters)) %>%
-    dm_add_tbl_impl(new_tbl, new_tbl_name_chr, list_of(new_filters)) %>%
+    dm_add_tbl_zoomed_impl(new_tbl, new_tbl_name_chr, list_of(new_filters)) %>%
     dm_get_def() %>%
     mutate(
       pks = if_else(table == new_tbl_name_chr, !!upd_pk, pks),
       fks = if_else(table == new_tbl_name_chr, !!upd_inc_fks, fks)
     ) %>%
-    new_dm3(zoomed = TRUE)
+    new_dm3(zoomed = TRUE, validate = FALSE)
 
   dm_wo_outgoing_fks %>%
     dm_insert_zoomed_outgoing_fks(new_tbl_name_chr, names_list$old_new_names[old_tbl_name], col_tracker_zoomed(dm)) %>%
     dm_clean_zoomed()
 }
 
+dm_select_tbl_zoomed_impl <- function(dm, selected) {
```

Comment:
Good catch that these two functions were needed to avoid producing an invalid dm at some point in `cdm_insert_zoomed_tbl()` (even though it somehow worked in the end).